src/main/java/com/ss/training/utopia/agent/dao/FlightDAO.java

```diff
@@ -12,5 +15,11 @@
 
 @Repository
 public interface FlightDAO extends JpaRepository<Flight, FlightPk> {
+    
+    @Query(
+        value = "SELECT * FROM tbl_flight f WHERE f.seatsAvailable > 0 AND f.departTime <= CURRENT_TIMESTAMP", 
```

Comment:
This `<=` should be `>` (or `>=` if we decide it's reasonable to let flights be booked the instant they leave).